static/sass/initial/_type.scss

```diff
@@ -57,30 +57,43 @@ h3 {
 a {
   #{$link} {
     @include text-underline('action');
-    color: inherit;
+    color: color('action');
     font-weight: bold;
     text-decoration: none;
     transition: color 150ms;
+
+    h1 &,
+    h2 &,
+    h3 & {
+      background-image: none; // remove link underline for headlines
+    }
   }
 
   #{$focus} {
-    color: color('action');
+    background-image: linear-gradient(color('action-hover'), color('action-hover'));
+    color: color('action-hover');
   }
 }
 
+/// @todo totally yucky, need a better class to hook into
+///   [data-module='rst-body'] is only for blog posts
+///   and no global inner-level class exists
```

Comment:
kinda. 
Goal: more top/bottom padding/margin on body-like content, but not for non-long form content.

If I make it on main, then there is too much space for certain content. See screenshots  for comparison. The blog page, if using the same margin/padding os the rst-body, is too spaced out
![too-much-space](https://cloud.githubusercontent.com/assets/1581694/22710421/d822c13a-ed42-11e6-8012-37a38469e445.jpg)

![just right](https://cloud.githubusercontent.com/assets/1581694/22710465/feea8de8-ed42-11e6-859f-f122506b6c58.jpg)